.github/workflows/discord.yml

```
@@ -0,0 +1,26 @@
+name: "discord-notify"
+on:
+  workflow_run:
+    workflows: [publish]
+    types:
+      - completed
+
+jobs:
+  notify-discord:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+
+      - name: Extract version
+        id: extract_version
+        uses: Saionaro/extract-package-version@v1.0.6
+
+      - name: Discord notification
+        env:
+          DISCORD_USERNAME: Subspace_Desktop-Releaser
```

```suggestion
          DISCORD_USERNAME: "Subspace Desktop Releases"
```
does this work?

Also could add `DISCORD_AVATAR` to add our Subspace Bot Icon.